controllers/blogController.js

```diff
@@ -0,0 +1,5 @@
+blogs_get = (req, res) => res.render("blog/index");
```

Comment:
I'm a little unclear on the difference between the home page and the blogs page. So, for now, both "/" and "/blogs" route to the same index.ejs.

Reply:
/ will be the home page and will show the latest blog posts and /blog/post_id will be the route for the individual posts.

Reply:
Ok right, got it. I'll be sure to add an :id parameter to the blog routes, then. Thanks Murfy!